src/site/content/en/handbook/inclusion-and-accessibility/index.md

```diff
@@ -5,20 +5,19 @@ authors:
   - mfriesenhahn
 date: 2019-06-26
 description: |
```

Comment:
I don't understand why we need an extra doc about this when we say we follow the google developer style guide. There's already a doc, and it has way more examples and explanation: https://developers.google.com/style/inclusive-documentation

Reply:
Yeah, that's a fair question. I think there are four general reasons:
1. The GDDSC includes things web.dev writers don't need to worry about since they're taken care of by the site itself. 
1. The GDDSC includes some red-line rules (e.g., [keep sentences shorter than 26 words](https://developers.google.com/style/accessibility#reading-ease)), which always give me pause, especially when rationales aren't provided.
1. The GDDSC _doesn't_ include some information I think is important (e.g., what reading level to target and how to know whether you're hitting it).
1. web.dev is deliberately departing from the GDDSC in certain respects to achieve the goals of the site (e.g., allowing culturally specific examples as long as essential info doesn't depend on them).

I _do_ think this section needs to be reframed though--and probably some content should be replaced with links to the GDDSC. Might be easier to talk that--and some of the other issues you've raised--through in real time?

Reply:
How do you even calculate reading levels in an article about software development? One metric of reading level formulas is counts of words with high numbers of syllables. Since software development is full of multi-syllable words, the results of any test are going to skew high. I could theoretically write a script that strips out the terms of art and calculates from the result. Ignoring that such a script would only be as good as its list of software terms, what would the result even mean?

By the way, which formula are you using?